index.cjs

```diff
@@ -747,19 +774,30 @@ safeql: try {
   }
 
   // Abort early if either of these modules are not installed
-  require.resolve('@ts-safeql/eslint-plugin');
-  require.resolve('dotenv-safe');
+  try {
+    require.resolve('@ts-safeql/eslint-plugin');
+    require.resolve('dotenv-safe');
+  } catch (error) {
+    throw new AddLineBreaksForEslintOutputInVscodeError(`Please reinstall the UpLeveled ESLint Config using the instructions on https://www.npmjs.com/package/eslint-config-upleveled
+${error}
+    `);
+  }
```

Comment:
I implemented a `try/catch` to capture the specific error message thrown by `require.resolve`